KafkaIngester/src/main/java/io/deephaven/kafka/ingest/JsonNodeBooleanFieldCopier.java

```diff
@@ -0,0 +1,51 @@
+package io.deephaven.kafka.ingest;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.deephaven.UncheckedDeephavenException;
+import io.deephaven.db.v2.sources.chunk.Attributes;
+import io.deephaven.db.v2.sources.chunk.ObjectChunk;
+import io.deephaven.db.v2.sources.chunk.WritableChunk;
+import io.deephaven.db.v2.sources.chunk.WritableByteChunk;
+import io.deephaven.util.BooleanUtils;
+
+public class JsonNodeBooleanFieldCopier implements FieldCopier {
+    private final String fieldName;
+
+    public JsonNodeBooleanFieldCopier(String fieldName) {
+        this.fieldName = fieldName;
+    }
+
+    @Override
+    public void copyField(ObjectChunk<Object, Attributes.Values> inputChunk, WritableChunk<Attributes.Values> publisherChunk, int sourceOffset, int destOffset, int length) {
+        final WritableByteChunk<Attributes.Values> output = publisherChunk.asWritableByteChunk();
+        for (int ii = 0; ii < length; ++ii) {
+            final JsonNode node = (JsonNode) inputChunk.get(ii + sourceOffset);
+            final String valueAsString = JsonNodeUtil.getString(node, fieldName, true, true);
+            final Boolean valueAsBoolean;
+            if (valueAsString == null) {
+                valueAsBoolean = null;
+            } else {
+                switch(valueAsString) {
```

Comment:
valueAsString.trim(); empty goes to null I think

Reply:
SG